.github/ISSUE_TEMPLATE/issue_form_template.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
```

Comment:
nobody will use this type of form, suggestion box and nurse's office are more than sufficient